README.md

```diff
@@ -6,10 +6,30 @@ This repository implements GFlowNets, generative flow networks for probabilistic
 
 ### pip
 
+Prereqs: install `fftw3`:
+
+```bash
+brew install fftw  # OSX
+sudo apt-get install fftw  # Ubuntu
+```
+
+To setup the environment with all dependencies, first install the pytorch `>=2.0.1` version appropriate to your system and some other dependencies more easily handled using Conda, and then pip install the package and remaining dependencies itself:
+
 ```bash
+conda install xtb-python -c conda-forge -y
+conda install pytorch torchvision torchaudio -c pytorch -y # Use the version appropriate to your environment.
+conda install pyshtools  -c conda-forge -y
```

Comment:
My anti-conda soul can't help but ask whether at least we could have a section about how to install things without conda, that is _normally_ :P I never use conda so I would argue that it is not really necessary.

Reply:
It's not required, no, but we need to think harder about how to get this set up on an arbitrary machine if we don't use it. As an exercise, try setting up this project locally on your laptop from scratch (with a clean python environment I mean).

On my machine it took about 3 hours of troubleshooting to ensure I had everything right.

Reply:
I set "everything" up recently in a new laptop and it was quite painless. However, with "everything" I am excluding some of the molecular modelling libraries that are a pain in the ass. I'll try to repeat this soon and document the process here.